### PR TITLE
uid updated from token emis username

### PR DESCRIPTION
**Pull request by sudeeppr1998**

### CODE REVIEW

File: src/services/telementryService.js [hunk 1]
```diff
@@ -153,19 +129,33 @@ export const error = (error, data) => {
...
+export const getEventOptions = () => {
+  var emis_username = 'anonymous'
+  if (localStorage.getItem('token') !== null) {
+    let jwtToken = localStorage.getItem('token');
+    var userDetails = jwt(jwtToken);
+    emis_username = userDetails.emis_username;
+  }
+
+  console.log(emis_username);
+  alert(emis_username);
```
Comment (gouravmore) [inline src/services/telementryService.js:141]: remove alert message

File: src/services/telementryService.js [hunk 2]
```diff
@@ -153,19 +129,33 @@ export const error = (error, data) => {
...
-      env: 'contentplayer',
-      uid: contextdata.uid,
+      pdata:{
+        // optional
+        id: process.env.REACT_APP_id, // Producer ID. For ex: For sunbird it would be "portal" or "genie"
+        ver: process.env.REACT_APP_ver, // Version of the App
+        pid: process.env.REACT_APP_pid, // Optional. In case the component is distributed, then which instance of that component
+      },
+      env: process.env.REACT_APP_env,
+      uid: emis_username,
```
Comment (gouravmore) [inline src/services/telementryService.js:152]: emis_username || ''anonymous"

File: src/services/telementryService.js (hunk 1 above)
Comment (sudeeppr1998) [reply src/services/telementryService.js]: Done

File: src/services/telementryService.js (hunk 2 above)
Comment (sudeeppr1998) [reply src/services/telementryService.js]: Done

**Pull request merged**